verticapy/tests/vModel/test_decision_tree_classifier.py

```diff
@@ -424,10 +424,10 @@ def test_score(self, model):
         )
         assert model.score(
             cutoff=0.9, method="npv", pos_label="Train"
-        ) == pytest.approx(0.0)
+        ) == pytest.approx(1.0)
```

Comment:
just double check if you really meant to change this to a different value.

Reply:
the npc was wrong. There was a huge mistake in the code. I've corrected it!